src/sinol_make/commands/run/__init__.py

```diff
@@ -717,24 +717,34 @@ def get_valid_input_files(self):
                 valid_input_files.append(test)
         return valid_input_files
 
+    def validate_existance_of_outputs(self):
+        """
+        Checks if all input files have corresponding output files.
+        """
+        valid_input_files = self.get_valid_input_files()
+        if len(valid_input_files) != len(self.tests):
+            missing_tests = list(set(self.tests) - set(valid_input_files))
+            missing_tests.sort()
+
+            print(util.warning('Missing output files for tests: ' + ', '.join(
+                [self.extract_file_name(test) for test in missing_tests])))
+            print(util.warning('Running only on tests with output files.'))
+            self.tests = valid_input_files
+            self.groups = self.get_groups(self.tests)
+
     def validate_tests(self):
```

Comment:
```suggestion
    def check_are_any_tests_to_run(self):
```
Sometimes it's as simple as shortening the description of the function.